task2/script.js

```diff
@@ -0,0 +1,105 @@
+'use strict';
+
+function isArrayOne(arr) {
+  return arr.constructor === Array;
+}
+
+function isArrayTwo(arr) {
+  return arr instanceof Array;
+}
+
+function isArrayThree(arr) {
+  return {}.toString.call(arr) === '[object Array]'
```

Comment:
are you sure it works?
`SyntaxError: expected expression, got '.'` is in my console